src/main/java/net/earthcomputer/clientcommands/command/KitCommand.java

```diff
@@ -0,0 +1,191 @@
+                .then(literal("create")
+                        .then(argument("name", StringArgumentType.string())
+                                .executes(ctx -> create(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("delete")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .executes(ctx -> delete(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("edit")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .executes(ctx -> edit(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("load")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .then(literal("--override")
+                                        .executes(ctx -> load(ctx.getSource(), StringArgumentType.getString(ctx, "name"), true)))
+                                .executes(ctx -> load(ctx.getSource(), StringArgumentType.getString(ctx, "name"), false))))
+                .then(literal("list")
+                        .executes(ctx -> list(ctx.getSource()))));
+    }
+
+    private static int create(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.get(name) != null) {
+            throw ALREADY_EXISTS_EXCEPTION.create(name);
+        }
+        kits.put(name, client.player.inventory.serialize(new ListTag()));
+        saveFile();
+        sendFeedback("commands.ckit.create.success", name);
+        return 0;
+    }
+
+    private static int delete(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.remove(name) == null) {
+            throw NOT_FOUND_EXCEPTION.create(name);
+        }
+        saveFile();
+        sendFeedback("commands.ckit.delete.success", name);
+        return 0;
+    }
+
+    private static int edit(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.get(name) == null) {
+            throw NOT_FOUND_EXCEPTION.create(name);
+        }
+        kits.put(name, client.player.inventory.serialize(new ListTag()));
+        saveFile();
+        sendFeedback("commands.ckit.edit.success", name);
+        return 0;
+    }
+
+    private static int load(ServerCommandSource source, String name, boolean override) throws CommandSyntaxException {
+        if (!client.player.isCreative() || !client.player.abilities.creativeMode) {
```

Comment:
Surely you don't need both of these conditions?

Reply:
Should I remove that?

Reply:
You need one of them, check which one you need